src/modules/common/components/CategoryListCaption.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import React from 'react'
+import styled, { type StyledComponent } from 'styled-components/native'
+import type { ThemeType } from '../../theme/constants/theme'
+
+const H1: StyledComponent<{}, ThemeType, *> = styled.Text`
+  padding: 20px 0;
+  font-size: 20px;
+  text-align: center;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.decorativeFontBold};
+`
+
+const CaptionWithThumbnail: StyledComponent<{}, ThemeType, *> = styled.Text`
+  padding: 0 0;
+  font-size: 20px;
+  text-align: center;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.decorativeFontBold};
```

Comment:
You could add `withThumbnail` to the PropsType of the styled component. Then you wouldn't have to define the quasi-same styled-component twice and instead use `padding: ${props => props.withThumbnail ? '0 0' : '20px 0'}`.